### PR TITLE
use only websockets for control channels

### PR DESCRIPTION
**Pull request by rmg**
* [x] enable WS control channel for dirvers
* [x] refactor/de-duplicate WS channel code in drivers

Depends on strongloop/strong-runner#11

### CODE REVIEW

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@slnode test please

**Comment by rmg** [comment]
@sam-github I don't want to hold things up any longer, and I'm pretty happy with the current state, so PTAL and we can merge this so you have less code to port to the drivers repo.

```
10 files changed, 183 insertions(+), 265 deletions(-)
```

File: lib/drivers/common/control.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var _ = require('lodash');
```
Comment (sam-github) [inline lib/drivers/common/control.js]: use strict

File: lib/drivers/docker/container.js [hunk 2]
```diff
@@ -274,51 +237,26 @@ Container.prototype.attachMonitor = function() {
   }
 };
 
-Container.prototype.updatePortMapping = function() {
-  this.debug('Docker::Container.updatePortMapping()');
+Container.prototype._resume = function() {
+  this.debug('Docker::Container._resume()');
```
Comment (sam-github) [inline lib/drivers/docker/container.js]: `'strong-pm:docker:container:` is already in the debug message, it doesn't have to be repeated. OK to keep it if you want, though.

**Comment by sam-github** [comment]
Looks much better, thanks. LGTM

File: lib/drivers/common/control.js (hunk 1 above)
Comment (rmg) [inline lib/drivers/common/control.js]: woops! thanks!

**Comment by rmg** [comment]
@slnode test please

**Pull request merged**